en/frames_rover/differential_rover.md

```diff
@@ -0,0 +1,112 @@
+
+   Select the **Apply and Restart** button.
+
+   ::: info
+   If this airframe does not show up in the UI, it can alternatively be selected by setting the [SYS_AUTOSTART](../advanced_config/parameter_reference.md#SYS_AUTOSTART) parameter to `50000`.
+   :::
+
+3. Open the [Actuators Configuration & Testing](../config/actuators.md) to map the motor functions to flight controller outputs.
+
+This is sufficient to drive the the rover in [manual mode](../flight_modes_rover/index.md#manual-mode) (see [Flight modes](../flight_modes_rover/index.md)).
+
+::: info
+The parameter [RD_MAN_YAW_SCALE](../advanced_config/parameter_reference.md#RD_MAN_YAW_SCALE) can be used to scale the manual input for the yaw rate.
+:::
+
+## Tuning (Basic)
+
+This section goes through the basic parameters that need to be set to use all other features for the differential-steering rover.
+Navigate to [Parameters](../advanced_config/parameters.md) in QGroundControl and set the following parameters:
+
+1. [RD_WHEEL_TRACK](../advanced_config/parameter_reference.md#RD_WHEEL_TRACK) [m]: Measure the distance from the center of the right wheel to the center of the left wheel.
+
+   ![Wheel track](../../assets/airframes/rover/rover_differential/wheel_track.png)
+
+2. [RD_MAX_SPEED](../advanced_config/parameter_reference.md#RD_MAX_SPEED) [m/s]: In manual mode, drive the rover with full throttle and enter the observed speed as the parameter.
+3. [RD_MAX_YAW_RATE](../advanced_config/parameter_reference.md#RD_MAX_YAW_RATE) [deg/s]: This is the maximum yaw rate you want to allow for your rover.
+  This will define the stick-to-yaw-rate mapping in acro mode as well as setting an upper limit for the yaw rate in mission mode.
+4. [RD_YAW_RATE_P](../advanced_config/parameter_reference.md#RD_YAW_RATE_P) and [RD_YAW_RATE_I](../advanced_config/parameter_reference.md#RD_YAW_RATE_I) [-]: Tuning parameters for the closed-loop yaw rate controller.
+
+   ::: info
+   This can be tuned by setting all previous parameters and then setting the rover to _acro mode_.
+   Use the right stick to yaw the rover on the spot and then observe the desired and actual yaw rate in the flight log.
+   Change parameters and iterate.
+
+   Suggestion: Start the tuning process with [RD_YAW_RATE_I](../advanced_config/parameter_reference.md#RD_YAW_RATE_I) equal to zero and only set if necessary.
+   :::
+
+This is enough to start using the rover in acro mode.
+To start driving mission the parameters in [Tuning (Mission)](#tuning-mission) also must be set.
+
+## Tuning (Mission)
+
+:::warning
+The parameters in [Tuning (Basic)](#tuning-basic) must also be set to drive missions!
+:::
+
+The module uses a control algorithm called pure pursuit, see [Mission Mode](../flight_modes_rover/index.md#mission-mode) for the basic tuning process.
+The additional parameters are separated into the following sections:
+
+### Mission Velocity
+
+These parameters tune velocity control in missions:
```

Comment:
This is what I thought they were for. Is it correct?

In line with the other comment, how do I work out how to set these values? I.e. instructional or rule of thumb guidance. This might be "just experiment - if the vehicle does X, Y, Z, then you need to lower this". Does that make sense?